docs/modules/darkMode.mjs

```diff
@@ -0,0 +1,16 @@
+const darkCheck = document.getElementById('night-light-checkbox');
+
+darkCheck.addEventListener('click', () => {
+  if (darkCheck.checked) {
+    document.body.classList.add('dark');
+    localStorage.setItem('animate-css', 'dark');
+  } else {
+    document.body.classList.remove('dark');
+    localStorage.removeItem('animate-css');
+  }
+})
+
+if (localStorage.getItem('animate-css')) {
+  document.body.className = 'dark';
+  darkCheck.checked = true;
+}
```

Comment:
Could you please wrap this whole file in a function and export it just like the other modules?